README.md

```diff
@@ -111,14 +111,81 @@ This function must return an object containing the same options given in paramet
 | `registryToken` | Registry Token. See [registry documentation](https://docs.cozy.io/en/cozy-stack/registry-publish/). Should not be mutated. |
 | `spaceName` | Space name in the Cozy registry. |
 
-##### `--registry-url <url>`
+
+#### `--postpublish <script_path>`
+
+Works exactly like the `prepublish` option, but will be executed after the publication.
+
+#### Multiple hooks and built-in hooks
+
+You can specify more than one hook by separating them with a `,`:
+
+```
+cozy-app-publish --prepublish <hook_name>, <other_hook_name>
+```
+
+Some hooks are shipped with cozy-app-publish and can be used by specifying their name:
+
+```
+cozy-app-publish --prepublish <builtin_hook_name>
+```
+
+##### Downcloud prepublish hook
+
+```
+cozy-app-publish --prepublish downcloud
+```
+
+This hook allows to upload the app to our downcloud server and sets the `appBuildUrl` accordingly.
+
+##### Rundeck postpublish hook
+
+```
+cozy-app-publish --prepublish rundeck
+```
+
+Deploys the app on rundeck. This hook requires several variables to be set as environment variables:
+
+- `RUNDECK_TOKEN`:  the token to use to interact with the Rundeck API
+- `TARGETS_DEV`: a comma-separated list of instances to deploy the app on, on the `dev` channel.
+- `TARGETS_BETA`: a comma-separated list of instances to deploy the app on, on the `beta` channel.
+- `TARGETS_STABLE`: a comma-separated list of instances to deploy the app on, on the `stable` channel.
```

Comment:
Would be cool if those environment variables were checked before the hook is run to fail faster. Each hook would need a `validate` function of some kind.